src/EnumSet.php

```diff
@@ -198,11 +202,76 @@ public function count()
         $cnt = 0;
         $ord = 0;
         do {
-            if ($this->bitset & (1 << $ord++)) {
+            if ($this->getBit($ord++)) {
                 ++$cnt;
             }
         } while ($ord !== $this->ordinalMax);
 
         return $cnt;
     }
+
+    /**
+     * Get the bitset
+     * @return int
+     */
+    public function getBitset() {
+        $bitset="";
+        foreach($this->bitset as $int) {
+            $bitset.=pack(
+                $this->integerSize==32?'V':'P',$int);
+        }
+        
+        return $bitset;
+    }
+
+    /**
+     * Set the bitset, it resets the current position of the iterator
+     * @param int $bitset
+     * @return void
+     */
+    public function setBitset($bitset) {
+        if(!is_string($bitset)) {
+            throw new \InvalidArgumentException("bitset must be a string");
```

Comment:
This exception is already imported and can be used without a full qualified class name